docs/topics/request-response.rst

```diff
@@ -339,6 +342,9 @@ fields with form data from :class:`Response` objects.
        .. versionadded:: 0.17
           The ``formxpath`` parameter.
 
+       .. versionadded:: 1.1.5
```

Comment:
We don't have this version yet :) I think this change will be included in 1.1.0.